custom-namespaces.yaml

```
@@ -571,6 +571,13 @@ firefox_accounts:
       type: table_view
       tables:
         - table: moz-fx-data-shared-prod.accounts_db.fxa_oauth_clients
+    monitor_db_counts:
+      type: table_view
+      tables:
+        - table: moz-fx-data-shared-prod.backend_derived.monitoring_db_counts_v1
```

The table reference is wrong, as the dataset should be `accounts_backend_derived`.  However, since @akkomar is adding a view specifically for this in mozilla/bigquery-etl#6599 we should wait until that gets deployed and use the view instead.
```suggestion
        - table: moz-fx-data-shared-prod.accounts_backend.monitoring_db_counts
```